src/v2/Components/Seo/LocalBusiness.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react"
+import { StructuredData } from "./StructuredData"
+import _ from "underscore"
```

Comment:
use lodash instead (we need a linter for this!)

Reply:
also I'd just pull out the relevant functions instead of importing _

Reply:
Damn I def searched the package.json file for `lowdash` and not finding it searched for `underscore` found it and picked that instead, haha. Will update!

Reply:
Also confused by this feedback @dzucconi 🤔 

<img width="654" alt="Screen Shot 2021-08-06 at 11 26 18 AM" src="https://user-images.githubusercontent.com/79799/128542060-345e1c5e-1a0e-44c4-90ec-84f3e1c9d928.png">

Reply:
Size concerns aside: we don't use underscore at all in the new app so importing it a single time will add the entire library unnecessarily. Theoretically lodash can be tree-shaken (hence advice importing just the functions you need) though I'm unclear as to whether or not we're reaping the benefit of that.

Reply:
`lowdash` !== `lodash`

Reply:
Fixed with d7b13a1 - is that what you were thinking?

Reply:
Exactly 👍